modules/basis/src/lib/parsers/parse-dds.js

```diff
@@ -0,0 +1,89 @@
+import {assert} from '@loaders.gl/core/';
```

Comment:
Was this copied from some open source code? Should we add some header?

Reply:
I don't see your point here.